features/hooks/filtering.feature

```diff
@@ -244,3 +244,186 @@ Feature: filters
       .after context
       """
 
+  Scenario: Filtering hooks using symbols
```

Comment:
Playing around with your PR locally, I'm seeing that this scenario is the only new one that requires your changes to pass.  Did you add the others just to flesh this cuke out with more examples or were you thinking they were affected by your changes?